data/multus/001-multus.yaml

```diff
@@ -134,11 +134,13 @@ spec:
               mountPath: /host/etc/cni/net.d
             - name: cnibin
               mountPath: /host/opt/cni/bin
+            - name: cnicache
+              mountPath: /host/var/lib/cni
           imagePullPolicy: {{ .ImagePullPolicy }}
           lifecycle:
             preStop:
               exec:
-                command: ["/bin/sh", "-c", "rm -f /host/etc/cni/net.d/00-multus.conf"]
+                command: ["/bin/sh", "-c", "rm -rf /host/etc/cni/net.d/00-multus.conf /host/var/lib/cni/*"]
```

Comment:
I think it would be safer to only delete stuff associated w/ multus; I find it strange that you're destroying the cache for all CNIs.

Did you attempt to only remove the stuff belonging to multus ? 

Something like (I did not check if this works, my goal is mainly to show my intention):
```yaml 
command:
  - "/bin/sh"
  - "-c"
  - "find /host/var/lib/cni -name multus* -exec rm -f {} ; && rm -f /host/etc/cni/net.d/00-multus.conf"
```

Reply:
I tried it and didn't work, maybe I can try with grep.